configs/etc/kubehound-reference.yaml

```diff
@@ -120,4 +120,14 @@ builder:
     # batch_size_small: 75
 
     #  # Cluster impact batch size for edge inserts
-    # batch_size_cluster_impact: 1
\ No newline at end of file
+    # batch_size_cluster_impact: 1
+  
+# Ingestor configuration (for KHaaS)
+# ingestor:
+#   bucket_name: "<your-bucket>"
+#   temp_dir: "/tmp/kubehound"
+#   archive_name: "archive.tar.gz"
+#   max_archive_size: 1073741824 # 1GB
+#   api: # GRPC endpoint for the ingestor
+#     url: "http://localhost:8080"
```

Comment:
Isn't it `http://localhost:9000` ? I know it is supposed to be changed by the user, but I believe we launch the ingestor with 9000 by default